internal/thelma/charts/mirror/mirror.go

```diff
@@ -157,7 +156,7 @@ func (m *mirror) fetchChart(chart ChartDefinition) error {
 		Prog: helm.ProgName,
 		Args: []string{
 			"fetch",
-			fmt.Sprintf(chart.Name),
+			chart.Name,
```

Comment:
since chart.Name is already a string and the there is no format specifier the wrapping function call does nothing